codecov.yml

```diff
@@ -0,0 +1,4 @@
+coverage:
+  range: 70..100
```

Comment:
What does this mean in practice?

Reply:
This defines the color range, with 70 displayed as bright red and 100 being pure green.